build-system/tasks/serve.js

```diff
@@ -96,7 +96,7 @@ gulp.task(
         'cache': '  Make local resources cacheable by the browser ' +
             '(default: false)',
         'inspect': '  Run nodemon in `inspect` mode',
-        'disable_dev_dashboard_cache': 'Disables the dev dashboard cache',
+        'enable_dev_dashboard_cache': '  Enables the dev dashboard cache',
```

Comment:
Can we mention this is enabled by default?

Reply:
Done.